qcengine/programs/entos.py

```diff
@@ -73,18 +67,35 @@ class EntosHarness(ProgramHarness):
     }
 
     # Available keywords for each of the energy commands
-    _dft_keywords: Set[str] = {"df"}
-    _xtb_keywords = {}
-    _keyword_map: Dict[str, Any] = {
-        "dft": _dft_keywords,
-        "xtb": _xtb_keywords
+    _dft_keywords_extra: Set[str] = {
+        "ansatz",
+        "df",
+        "orbital_grad_threshold",
+        "energy_threshold",
+        "coulomb_method",
+        "exchange_method"}
+    _hf_keywords_extra: Set[str] = {
+        "ansatz",
+        "df",
+        "orbital_grad_threshold",
+        "energy_threshold",
+        "coulomb_method",
+        "exchange_method"}
```

Comment:
There is redundancy between `_{xtb,dft,hf}_keywords_extra` and particularly `hf`/`dft`. Consider creating an inheritance-like structure of sets and supersets.